enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -75,7 +79,7 @@ func Register(env environment.Env) error {
 	return nil
 }
 
-func NewMigrationCache(migrationConfig *MigrationConfig, srcCache interfaces.Cache, destCache interfaces.Cache) *MigrationCache {
+func NewMigrationCache(migrationConfig *MigrationConfig, srcCache interfaces.Cache, destCache interfaces.Cache, srcShutdownFn func() error, destShutdownFn func() error) *MigrationCache {
```

Comment:
it would be slightly cleaner to define an interface for caches that can be stopped instead of passing the shutdown function separately. You can add a Stop() function to disk_cache that doesn't do anything to satisfy the interface.
```
type StoppableCache interface {
  interfaces.Cache
  Stop() error
}
```

Reply:
Done